x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/index.tsx

```diff
@@ -146,21 +146,26 @@ const getNetworkTopNFlowColumns = (
     truncateText: false,
     hideForMobile: false,
     render: ({ node }) => {
-      const attr =
+      const domainAttr =
+        networkTopNFlowType === NetworkTopNFlowType.source ? 'source.domain' : 'destination.domain';
+      const sourceIpAttr =
         networkTopNFlowType === NetworkTopNFlowType.source ? 'source.ip' : 'destination.ip';
-      const ip: string | null = get(attr, node);
+      const ip: string | null = get(sourceIpAttr, node);
+      const domain: string | null = get(domainAttr, node);
       if (ip != null) {
         return (
           <DraggableWrapper
             dataProvider={{
               and: [],
               enabled: true,
-              id: escapeDataProviderId(`networkTopNFlow-table-${networkTopNFlowType}-ip-${ip}`),
+              id: escapeDataProviderId(
+                `networkTopNFlow-table-${domain}-${networkTopNFlowType}-ip-${ip}`
+              ),
               name: ip,
               excluded: false,
               kqlQuery: '',
               queryMatch: {
-                field: attr,
+                field: domainAttr,
```

Comment:
I feel like we will do a search on the `source.ip` or `destination.ip` but not on the domain since we are showing the IP address of the source or destination